wp-config.php

```diff
@@ -9,10 +9,19 @@
  * directory.
  */
 
+// Provide a reference to the app root directory early.
+define( 'ROOT_DIR', __DIR__ );
```

Comment:
Should we namespace this? Maybe `HM\Platform\ROOT_DIR` or something?

Reply:
I think because it's the constant that is the _project_ root dir, having it in the NS might technically be correct but is less natural. We could call it `PROJECT_ROOT_DIR` to be more descriptive though.

Reply:
I'm thinking we should avoid global namespace where possible, both as a best practice and to avoid conflicts with WordPress.